packages/frontend/web/components/Footer.tsx

```diff
@@ -7,14 +7,24 @@ import {
     until,
     wide,
     desktop,
-    from,
 } from '@guardian/pasteup/breakpoints';
 import { textSans } from '@guardian/pasteup/typography';
+import { clearFix } from '@guardian/pasteup/mixins';
 
 import { Container } from '@guardian/guui';
+import { Pillars, pillarWidth, firstPillarWidth } from './Header/Nav/Pillars';
 import { palette } from '@guardian/pasteup/palette';
 import { ReaderRevenueLinks } from './Header/Nav/ReaderRevenueLinks';
 
+// CSS vars
+const emailSignupSideMargins = 10;
+const footerItemContainerPadding = 19;
+const emailSignupWidth =
```

Comment:
Calculating the the email signup width for the footer from the elements around it make for easily modified and consistent changes